robot_smach_states/src/robot_smach_states/manipulation/place.py

```diff
@@ -252,16 +249,19 @@ def __init__(self, robot, item_to_place, place_pose, arm, place_volume=None, upd
                                                     'failed': 'failed'}
                                        )
 
-            smach.StateMachine.add('PREPARE_PLACE', PreparePlace(robot, place_designator, arm),
-                                   transitions={'succeeded': 'NAVIGATE_TO_PLACE',
+            smach.StateMachine.add('PREPARE_PLACE', PreparePlace(robot, arm),
+                                   transitions={'succeeded': 'LOCK_DESIGNATOR',
                                                 'failed': 'failed'})
 
-            smach.StateMachine.add('NAVIGATE_TO_PLACE', NavigateToPlace(robot, place_designator, arm),
+            smach.StateMachine.add('LOCK_DESIGNATOR', LockDesignator(locking_place_designator),
```

Comment:
Should we unlock it afterwards or not?

Reply:
We don't need to. The locking designator only exists within the scope of Place. After place has been executed we no longer need it.